### DISCUSSION TITLE
Display_field in custom scripts. Netbox v3.x.x

### DISCUSSION THREAD

**Discussion by AlekseyMamedoff** [netbox-community/netbox#10047] [Q&A]
Hi!
"Display_field" has gone from Netbox v3. Can anyone show me how this functional use for now. 
It would be nice to see example of code.
it should be simple I understand but i searched for two days and got nothing:(

**Comment by bogdancordos** [comment]
Can you provide a sample of your script that uses "display_field"?

**Reply by AlekseyMamedoff** [reply]
```
from ipam.models import IPAddress
from extras.scripts import *

class Test(Script):
    class Meta:
        name = "Test"

    ip = ObjectVar(
        label='Service',
        model=IPAddress,
        required=False,
#        display_field='description', # we got exception if uncomment it
        )

    def run(self, data, commit):
        return data['ip'].description

```
When we try to execute this script it show the ip addresses itself in drop-down menu i want to see it's description instead. Before Netbox v3 it works with "display_field".

![Screenshot from 2022-08-22 12-33-45](https://user-images.githubusercontent.com/51682208/185891203-70ad62ac-3134-498f-a5ce-c78586fd1a9f.png)

**Comment by bogdancordos** [comment]
As per 3.0.0 release notes, `display_field` has been removed, try using `display` instead.

**Reply by AlekseyMamedoff** [reply]
It was the first i try after upgrade:)
Code like this:

```
    ip = ObjectVar(
        label='Service',
        model=IPAddress,
        required=False,
        display='description', 
        )
```

 It generate exception:
```
<class 'TypeError'>

__init__() got an unexpected keyword argument 'display'

Python version: 3.8.10
NetBox version: 3.3-beta2
```

We use Netbox v3.2.7 as main version. It has same problem.

**Reply by markkuleinio** [reply]
I don't use custom scripts, but I suppose you should use field name `description` instead.

**Reply by craized** [reply]
I'm having the same problem using custom scripts for NetBox IPAM operations. I need to list my Prefix description field, not the prefixes themselves. I was disappointed to see that the display_field option was removed.
